### PR TITLE
Split branding style themes

### PR DESCRIPTION
**Pull request by ncounter**
## What does this PR change?

Depending on the `product_name` value from the `Configs` it makes usage of different styles. At the moment it is a complete copy and fork of the full style stack, a proper cleanup and sharing what can be shared should happen but it can be postponed in a separate PR later.

## GUI diff

SUSE Manager:
Screenshot from 2020-05-20 01-55-37

Uyuni:
Screenshot from 2020-05-20 01-59-50

- [x] **DONE**

## Documentation
- No documentation needed: **add explanation. This can't be used if there is a GUI diff**
- [doc-susemanager](https://github.com/SUSE/doc-susemanager) PR or issue was created (GitHub automatic link expected below)

- [x] **DONE**

## Test coverage
- No tests: nothing to test

- [x] **DONE**

## Links

Fixes #

Relevant branches (GitHub automatic links expected below):
 - Manager

- [ ] **DONE**

### CODE REVIEW

File: java/code/webapp/WEB-INF/decorators/layout_head.jsp [hunk 1]
```diff
@@ -37,14 +39,28 @@
...
-    <rhn:require acl="is(development_environment)">
-      <link rel="stylesheet/less" type="text/css" href="/css/spacewalk.less" />
-      <script>less = { env: 'development' };</script>
-      <script src="/javascript/less.js"></script>
-    </rhn:require>
-    <rhn:require acl="not is(development_environment)">
-      <link rel="stylesheet" href="/css/spacewalk.css?cb=${cb_version}" />
-    </rhn:require>
+    <c:choose>
+        <c:when test='${Config.get().getString("product_name").compareToIgnoreCase("Uyuni") == 0}'>
```
Comment (mcalmer) [inline java/code/webapp/WEB-INF/decorators/layout_head.jsp]: The test == 0 means "true" ? Are you sure?

**Review by mcalmer — CHANGES_REQUESTED** [review]
There is one place where I have doubt

File: java/code/webapp/WEB-INF/decorators/layout_head.jsp (hunk 1 above)
Comment (ncounter) [reply java/code/webapp/WEB-INF/decorators/layout_head.jsp]: Well, I have to admit that I just copy-pasted, but giving it a manual try it worked. Anyway, I took it from [here](https://github.com/uyuni-project/uyuni/blob/master/java/code/webapp/WEB-INF/includes/leftnav.jsp#L8) and AFAIK it is working as expected in the current setup. Am I overlooking something?
I will log the value of the test anyway to double check if this is the proper way of testing it, or there is a better/more robust check function we can use.
Thanks for the heads up

Comment (ncounter) [reply java/code/webapp/WEB-INF/decorators/layout_head.jsp]: Anyway, based on your point that was good enough to raise awareness the check was not really good shaped, I moved to the better method it already exists in `ConfigDefaults.java`. This looks much better now :slightly_smiling_face:

Comment (mcalmer) [reply java/code/webapp/WEB-INF/decorators/layout_head.jsp]: Hmm, we call a "compare" function. They may give -1, 0, 1 for lower, equal, greater.
That makes sense

**Review by mcalmer — APPROVED** [review]

**Comment by ncounter** [comment]
Merged into https://github.com/uyuni-project/uyuni/pull/2133

**Pull request closed without merging**